app/assets/stylesheets/components/select/_selector.scss

```diff
@@ -0,0 +1,75 @@
+.selector {
+    border: rem-calc(1) solid $grey-xlight;
+    border-radius: 0;
+    
+    position: relative;
+    z-index: 100;
+    
+    display: block;
+
+    &__selected {
+        cursor: default;
+
+        display: flex;
+        align-items: center;
+        justify-content: space-between;
+
+        &:focus {
+            box-shadow: 0 0 0 rem-calc(2) rgba($grey-light, 0.25);
+        }
+    }
```

Comment:
I personally think it's a bit easier to read with a break line between the classes.